app/assets/javascripts/discourse/app/widgets/user-menu.js

```diff
@@ -31,6 +31,7 @@ createWidget("user-menu-links", {
       href: `${this.attrs.path}/summary`,
       action: UserMenuAction.QUICK_ACCESS,
       actionParam: QuickAccess.PROFILE,
+      alt: "user preferences icon",
```

Comment:
I agree with @romanrizzi, these should be translatable. Do we have to specify "user ... icon" here? Since these are already in a user menu context, maybe just the label for the action (preferences, bookmarks, etc.) can be used?

Reply:
Ah yes, translations, as well as your idea of shortening to just the name of the tab is a good idea. 👍